drivers/pwm/pwm_stm32.c

```diff
@@ -720,8 +721,6 @@ replaced by 'st,prescaler' property in parent node, aka timers"
 									       \
 	static const struct pwm_stm32_config pwm_stm32_config_##index = {      \
 		.timer = (TIM_TypeDef *)DT_REG_ADDR(DT_INST_PARENT(index)),    \
-		/* For compatibility reason, use pwm st_prescaler property  */ \
-		/* if exist, otherwise use parent (timers) property         */ \
```

Comment:
Is that required to clean this comment up?
If really required, can it be put elsewhere?

Reply:
forgot to undo this one (clang-format gets confused with it). btw, shouldn't the `st,prescaler` at pwm level removed?

Reply:
Indeed, @ABOSTM can you check if it's now time to remove this?

Reply:
Yes it is time to clean up,
we will take care of it.